### DISCUSSION TITLE
when will dall-e models be included ?

### DISCUSSION THREAD

**Discussion by younes-io** [ChatGPTNextWeb/NextChat#3755] [Q&A] [answered]
I'd like to know if it is expected to have DALL-E models included as well ? If so, is there a timeline ? Thank you!

**Comment by lloydzhou** [comment, marked as answer]
现在支持两种模型：
1. dall-e-3作为openai下面的模型直接使用
2. Dalle作为插件在v2.15.0版本使用（这里是demo: https://github.com/ChatGPTNextWeb/NextChat-Awesome-Plugins/tree/main/plugins/dalle）
